Sources/GoodNetworking/Session/Logger/LoggingEventMonitor.swift

```diff
@@ -38,7 +38,12 @@ public class LoggingEventMonitor: EventMonitor {
         } else {
             nil
         }
-        let responseBodyMessage = parse(data: response.data, error: response.error as NSError?, prefix: "⬇️ Response body:")
+        
+        let responseBodyMessage = if Self.responseTypeWhiteList.contains(where: { $0 == response.response?.mimeType }) {
```

Comment:
Nechceme tu pridat moznost vypnut to uplne ? :) 

Neda mi pozerat sa nato z takeho pohladu ze ked pridam limitaciu pridam aj mozno vypnut ju. Ako v linte :)